chapters/remote/remote-flows.md

```diff
@@ -0,0 +1,71 @@
+# Remote Flows with JinaD
+
+### Prerequisites
+
+Before you start, make sure you have read the [prerequisites for using JinaD](https://docs.jina.ai/chapters/remote/jinad.html#prerequisites).
+
+## REST API
+
+[Refer the detailed API reference for JinaD](https://api.jina.ai/daemon/#tag/flows)
+
+A common use case is running the Flow on the remote with pods distributed across different machines.
+
+#### Examples
+
+Define `flow.yml` with the correct `JinaD` host details.
+
+```yaml
+jtype: Flow
+pods:
+    - uses: _logforward
+      host: 1.2.3.4:8000
+      upload-files:
+        - custom_encoder_1.yml
+    - uses: _logforward
+      host: 2.3.4.5:8000
+      upload-files:
+        - custom_encoder_2.yml
+```
+
+###### 1. Create a Flow
+
+`POST /flows` endpoint accepts a flow yaml file and returns an id to track the Flow.
```

Comment:
is there a standard for documenting API endpoints?

Would be nice to have more structure here, maybe a table to describe the parameters the user has to pass in the POST and the response format

Reply:
We are pointing the users [here ](https://api.jina.ai/daemon/) for now. This would be done if we have a good way to host the api description inside sphinx itself.